plasticos_web_leads/models/web_lead.py

```
@@ -0,0 +1,427 @@
+        intake = Intake.create(intake_vals)
+        return intake
+
+    # ═════════════════════════════════════════════════════════
+    # Manual Actions
+    # ═════════════════════════════════════════════════════════
+
+    def action_retry_processing(self):
+        """Retry processing a lead that errored."""
+        for rec in self:
+            if rec.state != "error":
+                raise UserError("Only errored leads can be retried.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)